signalfx-yammer/src/main/java/com/signalfx/codahale/reporter/SfUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.signalfx.codahale.reporter;
+
+import com.yammer.metrics.core.Gauge;
```

Comment:
No, method removeMetrics not supported, uses different library.
I can't see duplicated